src/components/plot/index.tsx

```diff
@@ -341,4 +350,11 @@ export class PlotBase extends React.PureComponent<PlotProps, PlotState> {
   }
 }
 
-export const Plot = CSSModules(PlotBase, styles);
+
+export const Plot = connect<PlotConnectProps, {}, PlotOwnProps>(
+  (state: State) => {
+    return {
+      voyagerConfig: selectConfig(state)
```

Comment:
Plot should be a dumb (renderer-only) component, so it should not use `connect`.   
Instead, `config` should be passed from its parent component, which should already contain `connect`.

Let's just name it `config` too as config in this project (except in `<VegaLite/>`)  should mean Voyager's config anyway.

Reply:
ok makes sense if we want to extract this as a separate component (or separate repo).